stablehlo/tests/infer_stablehlo.mlir

```diff
@@ -1508,9 +1508,9 @@ func.func @real_dynamic_slice(%arg0: tensor<?xf32>, %arg1: tensor<1xindex>, %arg
 
 // -----
 
-// CHECK-LABEL: func @dot_general
+// CHECK-LABEL: func @dot_general_c12
```

Comment:
Let's keep `dot_general` given that I don't think we'll be distinguishing test cases in this file by constraint names.

Reply:
The current convention in the [checklist 6.i](https://github.com/openxla/stablehlo/blob/main/docs/reference_checklist.md) is to also tag the tests with constraints in this file. There are other ops whose constraints do matter (mainly batch_norm_* ops). If we decide to remove it, it's probably best to address them in bulk.